Tweaks

alhankeser opened the pull request:
Resolves #16 

![Screen Shot 2019-12-11 at 23 51 10](https://user-images.githubusercontent.com/15135669/70683997-86b3c800-1c72-11ea-951e-7863d3310fed.png)

alhankeser committed:
docs(reporting): improve google analytics screenshots and add outcome

alhankeser committed:
fix(reporting): include correct image

alhankeser committed:
style(branding): fix heading spacing, logo and simplify docs home

alhankeser committed:
refactor(sidebar): create consistent organization between react and vue

will need constant maintenance; need to create an outline of the complete version so we can build
off of that.

foxted commented on .vuepress/theme/components/Home.vue:
```diff
@@ -1,14 +1,14 @@
-            <div class="heroBackground"><span></span><span></span><span></span></div>
-            <div class="header relative p-0 z-10">
+        <header class="relative flex flex-col flex-grow justify-center py-8">
+            <!-- <div class="heroBackground"><span></span><span></span><span></span></div> -->
```
@alhankeser Decided to remove the background?

foxted committed:
chore: solve security vulnerability

there is one remaining, but it's upstream (vue-server-renderer needs to be updated first)
